tests/cy.js

```diff
@@ -1,35 +1,41 @@
 module.exports = {
-	'35722123456': '+35722123456',
-	'22123456': '+35722123456',
-	'+35799123456': '+35799123456',
-	'35799123456': '+35799123456',
-	'99123456': '+35799123456',
-	'+357 22 123456': '+35722123456',
-	'357 22 123456': '+35722123456',
-	'22 123456': '+35722123456',
-	'(+357)22123456': '+35722123456',
-	'(357)22123456': '+35722123456',
-	'(+357) 221 23 456': '+35722123456',
-	'(357) 221 23 456': '+35722123456',
-	'(+35722)123456': '+35722123456',
-	'(35722)123456': '+35722123456',
-	'(+357 22) 123 456': '+35722123456',
-	'(357 22) 123 456': '+35722123456',
-	'(22) 123 456': '+35722123456',
-	'(+357) 22-123-456': '+35722123456',
-	'(357) 22-123-456': '+35722123456',
-	'22-123-456': '+35722123456',
-	'+357-22-123456': '+35722123456',
-	'357-22-123456': '+35722123456',
-	'+357 (22) 123-456': '+35722123456',
-	'357 (22) 123-456': '+35722123456',
-	'(22) 123-456': '+35722123456',
-	'+357(22)123456': '+35722123456',
-	'357(22)123456': '+35722123456',
-	'(22)123456': '+35722123456',
-	'+357 (221) 23 456': '+35722123456',
-	'357 (221) 23 456': '+35722123456',
-	'(221) 23 456': '+35722123456',
-	'+35122123456': null,
-	'35122123456': null
+	// '35722123456': '+35722123456',
```

Comment:
if it's not used anymore, please delete code, we can restore from history, no need to have dead code commented